website/docs/advanced-topics/struct-components/lifecycle.mdx

```diff
@@ -145,7 +145,7 @@ Note that this lifecycle method does not require an implementation and will do n
 Communication with components happens primarily through messages which are handled by the
 `update` lifecycle method. This allows the component to update itself
 based on what the message was, and determine if it needs to re-render itself. Messages can be sent
-by event listeners, child components, Agents, Services, or Futures.
+by event listeners, child and parent components, Agents, Services, or Futures.
```

Comment:
```suggestion
from any callback before a component is destroyed.
```

Not sure what's the better way to phrase this, but services has been removed and a message can be sent essentially from anywhere at anytime before a component is destroyed (given one can obtain the `Scope`).